.github/workflows/tests.yml

```diff
@@ -4,14 +4,17 @@ name: tests
 
 on:
   push:
-    branches: [ master ]
   pull_request:
   schedule:
     - cron: '0 12 * * 0'  # run once a week on Sunday
+  # Allow to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
@jugmac00 This line was needed to be able to re-run the job via the web-UI. It was missing so I was not able to rerun CI without creating a new commit.

Reply:
@jugmac00 Interesting you found a hidden switch to re-run the tests without this line.